pom.xml

```diff
@@ -24,8 +24,9 @@
     <stack.version>3.5.0-SNAPSHOT</stack.version>
     <junit.version>4.12</junit.version>
     <paho.version>1.1.0</paho.version>
+    <fabric8-docker-plugin.version>0.22.1</fabric8-docker-plugin.version>
```

Comment:
Out of curiosity. I'm ok with that but do you have increased the version for any specific reason ?

Reply:
No specific reason, just a newest version